src/blocks.zig

```diff
@@ -69,6 +69,13 @@ var _alwaysViewThrough: [maxBlockCount]bool = undefined;
 var _hasBackFace: [maxBlockCount]bool = undefined;
 var _blockTags: [maxBlockCount][]Tag = undefined;
 var _light: [maxBlockCount]u32 = undefined;
+
+/// is this block decayable?
+var _onBreak: [maxBlockCount]ServerBlockCallback = undefined;
+var _onUpdate: [maxBlockCount]ServerBlockCallback = undefined;
+var _decayProhibitor: [maxBlockCount]bool = undefined;
+var _decayReplacement: [maxBlockCount]u16 = undefined;
```

Comment:
The replacement should be part of the callback data, since it isn't needed for everything. Also it should be a full block, and not just the type (this would allow for some sort of progressive decay)